django/utils/functional.py

```diff
@@ -50,6 +50,31 @@ def __get__(self, instance, cls=None):
 
 
 class classproperty:
+    """
+    Like @property but allows the property to be accessed directly from the class
+    without requiring an instance of the class, example usage::
```

Comment:
I don't see much value in adding an example in this docstring. Please remove it.